external-task-worker/index.js

```diff
@@ -0,0 +1,58 @@
+const {HttpClient} = require('@essential-projects/http');
+const {
+  ExternalTaskApiClientService,
+  ExternalTaskApiExternalAccessor,
+  ExternalTaskWorker,
+} = require('@process-engine/external_task_api_client');
+
+const {
+  ExternalTaskFinished,
+} = require('@process-engine/external_task_api_contracts');
```

Comment:
Benutz besser `@process-engine/consumer_api_contracts`.

`@process-engine/external_task_api_contracts` ist seit längerer Zeit schon als deprecated markiert und wird nicht mehr gepflegt.

```suggestion
} = require('@process-engine/consumer_api_contracts');
```